scripts/lib/common.sh

```diff
@@ -32,15 +34,18 @@ if [[ $REGION == "cn-north-1" || $REGION == "cn-northwest-1" ]]; then
   go env -w GOSUMDB=sum.golang.google.cn
 fi
 
-GET_CLUSTER_INFO_CMD="aws eks describe-cluster --name $CLUSTER_NAME --region $REGION"
+if [[ $USE_EKSCTL == true ]]; then
+  GET_CLUSTER_INFO_CMD="aws eks describe-cluster --name $CLUSTER_NAME --region $REGION"
 
-if [[ -z "${ENDPOINT}" ]]; then
-  CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD)
-else
-  CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD --endpoint $ENDPOINT)
+  if [[ -z "${ENDPOINT}" ]]; then
+    CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD)
+  else
+    CLUSTER_INFO=$($GET_CLUSTER_INFO_CMD --endpoint $ENDPOINT)
+  fi
+
+  VPC_ID=$(echo $CLUSTER_INFO | jq -r '.cluster.resourcesVpcConfig.vpcId')
+
+  echo "VPC ID: $VPC_ID"
```

Comment:
Are these not required for kops?

Reply:
I didn't quite catch that.. VPC_ID is required by ginkgo tests which we pass as a parameter